src/platform/site-wide/sass/modules/_m-homepage.scss

```diff
@@ -5,16 +5,75 @@
 .hub-links-list {
   margin-bottom: 0;
   padding-right: 5px;
+  padding-left: 1rem;
+  list-style: none;
 }
 
-#homepage-hub, .va-sidebarnav, .vet-toolbar {
+.homepage-hub, .va-sidebarnav, .vet-toolbar {
   [href^="http"]:not(.usa-button-primary), [rel="external"] {
     background-image: none;
     padding-right: 0px;
 
   }
 }
 
+.homepage-hub {
+  @include linear-gradient-background($color-old-browser-background-start, $color-old-browser-background-end);
+  padding-top: 1.5em;
+
+  @include media($medium-large-screen) {
+    padding-top: 3em;
+  }
+}
+
+.homepage-hub-container {
+  max-width: 100rem;
+
+  @include media($medium-screen) {
+    margin: auto;
+    width: 85%;
+  }
+
+  @include media($large-screen) {
+    margin: auto;
+    width: 64%;
+  }
+}
+
+.homepage-heading {
+  color: $color-white;
+  text-align: center;
+  margin-top: 0;
+  margin-bottom: 1em;
+
+  @include media($medium-large-screen) {
+    margin-bottom: 1.5em;
+  }
+}
+
+.hub-links {
+  &-row {
+    display: flex;
```

Comment:
I changed these cards to use flexbox instead of the grid. The grid had gutters that are percentage based, which we can't match with the bottom margin. And making them a fixed size doesn't play all that well with the grid. The flexbox approach is fairly simple with a fixed margin.